spec/factories/activity.rb

```diff
@@ -0,0 +1,15 @@
+FactoryBot.define do
+  factory :activity do
+    title { Faker::Lorem.sentence }
+    identifier { "GB-GOV-13-GCRF" }
```

Comment:
We'll be making a lot of activities so worth making this dynamic so they are unique, something like:

```
sequence(:identifier) {|n| "GB-GOV-13-GCRF-#{n}" }
```